src/main/java/seedu/address/logic/commands/UnmarkAttendanceCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Unmarks the attendance of a particular student.
+ */
+public class UnmarkAttendanceCommand extends Command {
+
+    public static final String COMMAND_WORD = "unmark";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Unmarks the attendance of a particular student.\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_SUCCESS = "Attendance unmarked successfully.";
+
+    private final Index targetIndex;
```

Comment:
Possibly able to make `unmark` a bulk command like delete for further ease of use, like `unmark 1, 2, 10, 11`